slack_sdk/web/async_client.py

```diff
@@ -76,7 +76,12 @@ class AsyncWebClient(AsyncBaseClient):
     """
 
     async def admin_analytics_getFile(
-        self, *, date: str, type: str, **kwargs
+        self,
+        *,
+        type: str,
+        date: Optional[str] = None,
```

Comment:
The `date` used to be required when we had only `member` type. Now it can be optional in the case where an API call has `public_channel` type along with `metadata_only`.